components/form/introduction.rst

```diff
@@ -697,6 +697,13 @@ method to access the list of errors. It returns a
     This is useful, for example, if you want to use PHP's ``array_`` function
     on the form errors.
 
+Clearing Form Errors
+~~~~~~~~~~~~~~~~~~~~
+
+Any errors can be manually cleared using the :method:`Symfony\\Component\\Form\\FormInterface::clearErrors`
+method. This is useful when you'd like to validate the form without showing validation errors to the user
+(i.e. during a partial AJAX submission or :ref:`dynamic form modification <cookbook-form-dynamic-form-modification>`).
```

Comment:
Could anyone please provide some guidance on how I'd link to [this cookbook article](http://symfony.com/doc/current/cookbook/form/dynamic_form_modification.html)?  Apparently I've done it wrong :-/